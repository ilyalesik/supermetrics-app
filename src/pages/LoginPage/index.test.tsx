import renderer from "react-test-renderer";
import React from "react";
import { LoginPage } from "./index";

it("renders LoginPage", () => {
  const tree = renderer
    .create(
      <LoginPage>
        <div>form</div>
      </LoginPage>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
