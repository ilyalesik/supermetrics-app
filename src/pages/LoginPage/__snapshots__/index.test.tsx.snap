// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LoginPage 1`] = `
<div
  className="login-page-container"
>
  <div>
    form
  </div>
</div>
`;
